src/sidebar.js

```diff
@@ -134,7 +134,6 @@ module.exports = {
             'kratos/quickstart',
             'kratos/install',
             'kratos/configuring',
-            'kratos/contributing'
```

Comment:
I think you should add redirects for the pages that get removed in the https://github.com/ory/docs/blob/master/vercel.json file. That's what I was asked to do when I removed/moved docs around ;)

Reply:
I added them here: 
https://github.com/ory/web/pull/613

But I was also not sure if I should use the vercel.json or the _redirects in ory/web. 
I guess it does not matter? 
(other than us getting confused because there is so many redirects and two files).
 
Maybe we should do all /docs redirects in the vercel.json and the rest in ory/web?

Happy to add it to vercel.json, but lets come up with a rule so its clear where to add which redirect 🙏

Reply:
Oh man, I'm not sure what's the convention. If it were up to me I'd stick to handling redirects for the documentation in this repo and limit the other file to website-related redirects only. I'm not sure why everything's structured the way it is now.

Reply:
Ok I will add the redirects to the vercel.json here and then we handle docs redirects there going forward. 
We should then also move the other redirects for docs here.